pytest_fixtures/component/taxonomy.py

```diff
@@ -79,6 +79,7 @@ def module_org_with_manifest(module_org, module_target_sat):
 def module_entitlement_manifest_org(module_org, module_entitlement_manifest, module_target_sat):
     """Creates an organization and uploads an entitlement mode manifest generated with manifester"""
     module_target_sat.upload_manifest(module_org.id, module_entitlement_manifest.content)
+    module_org.sca_disable()
```

Comment:
`Satellite will throw a 400 response if we attempt to change from SCA mode to entitlement mode before a manifest has been uploaded to an org`, @synkd so is this expected or not?

@rplevka has reported a BZ for this behavior https://bugzilla.redhat.com/show_bug.cgi?id=2159974

Reply:
I think this is likely a bug after reading @rplevka's description, although it did not occur to me previously that this was the case.

Reply:
@synkd Thanks for confirming.